simulator/src/main/java/com/hazelcast/simulator/worker/TestContainer.java

```diff
@@ -152,10 +152,15 @@ public T getTestContext() {
         return testContext;
     }
 
-    public long getOperationCount() throws Exception {
-        Long count = invokeMethod((operationCountWorkerInstance != null) ? operationCountWorkerInstance : testClassInstance,
-                operationCountMethod);
-        return (count == null ? -1 : count);
+    public long getOperationCount() {
+        try {
+            Long count = invokeMethod((operationCountWorkerInstance != null) ? operationCountWorkerInstance : testClassInstance,
+                    operationCountMethod);
+            return (count == null ? -1 : count);
+        } catch (Exception e) {
+            LOGGER.warn("Exception while retrieving operation count from " + testCase.getId() + ": " + e.getMessage());
```

Comment:
won't this produce awful lot of warnings when the method is not available? I thought you would log it to `finest` - disabled by default, but with an option to enable it if needed.